mango-core/src/main/scala/org/bdgenomics/mango/models/TrackedLayout.scala

```diff
@@ -57,6 +59,27 @@ object TrackedLayout {
     val ref2 = rec2._1
     ref1.overlaps(ref2)
   }
+
+  def overlaps[T](rec1: (ReferenceRegion, T), recs2: Seq[(ReferenceRegion, T)]): Boolean = {
+    val ref1 = rec1._1
+    val recs2List = recs2.toList
```

Comment:
I don't think you need to make this a `List`. The `min` and `max` operations should exist on `Seq`.